### ISSUE TITLE
Unable to export csv when using union select

### ISSUE THREAD

**Issue by lukasg** [phpmyadmin/phpmyadmin#12619]
### Steps to reproduce
1) Run a query with union selects, in my case:
```
(SELECT p.project_no, p.name, c.name as client,
p.total_revenue/k.rate_one_eur as total_revenue_EUR ,
p.revenue/k.rate_one_eur as sp_revenue_EUR , p.description as overview,
p.objective as objectives, p.activities
FROM consultancy_projects p, contacts c, currencies_beta k LIMIT 1000)
UNION
(SELECT p.project_no, p.name, c.name as client,
p.total_revenue/k.rate_one_eur as total_revenue_EUR ,
p.revenue/k.rate_one_eur as sp_revenue_EUR , p.description as overview,
p.objective as objectives, p.activities
FROM consultancy_projects p, contacts c, currencies_beta k LIMIT 1000);
```
2) After you get the result set click on 'Export' and choose 'CSV'. (Actually it doesn't matter, it's the same problem for all export formats)

### Expected behaviour
I should get the resulting CSV with the correct result set.

### Actual behaviour
I get a CSV file full of errors:

```
<div class="error"><h1>Error</h1><p><strong>Static analysis:</strong></p><p>2 errors were found during analysis.</p><p><ol><li>This type of clause was previously parsed. (near "FROM" at position 455)</li><li>Unrecognized statement type. (near "FROM" at position 455)</li></ol></p><p><strong>SQL query:</strong>
<a href="./url.php?url=https%3A%2F%2Fdev.mysql.com%2Fdoc%2Frefman%2F5.5%2Fen%2Fselect.html" target="mysql_doc"><img src="themes/dot.gif" title="Documentation" alt="Documentation" class="icon ic_b_help" /></a><a href="tbl_sql.php?sql_query=%28SELECT+consultancy_projects.project_no%2C+consultancy_projects.name%2C+contacts.name+as+client%2C%0D%0Aconsultancy_projects.total_revenue%2Fcurrencies_beta.rate_one_eur+as+total_revenue_EUR+%2C%0D%0Aconsultancy_projects.revenue%2Fcurrencies_beta.rate_one_eur+as+sp_revenue_EUR+%2C+consultancy_projects.description+as+overview%2C%0D%0Aconsultancy_projects.objective+as+objectives%2C+consultancy_projects.activities%0D%0AFROM+consultancy_projects+p%2C+contacts+c%2C+currencies_beta+k+LIMIT+10%29+FROM+%60consultancy_projects%60%2C+%60contacts%60%2C+%60currencies_beta%60+UNION%0D%0A%28SELECT+consultancy_projects.project_no%2C+consultancy_projects.name%2C+contacts.name+as+client%2C%0D%0Aconsultancy_projects.total_revenue%2Fcurrencies_beta.rate_one_eur+as+total_revenue_EUR+%2C%0D%0Aconsultancy_projects.revenue%2Fcurrencies_beta.rate_one_eur+as+sp_revenue_EUR+%2C+consultancy_projects.description+as+overview%2C%0D%0Aconsultancy_projects.objective+as+objectives%2C+consultancy_projects.activities%0D%0AFROM+consultancy_projects+p%2C+contacts+c%2C+currencies_beta+k+LIMIT+10%29&amp;show_query=1&amp;db=southpo_pm&amp;table=currencies_beta&amp;token=3496c2fe0e374e0a16cfc5d4c78887d1"><span class="nowrap"><img src="themes/dot.gif" title="Edit" alt="Edit" class="icon ic_b_edit" />&nbsp;Edit</span></a>    </p>
<p>
(SELECT consultancy_projects.project_no, consultancy_projects.name, contacts.name as client,
consultancy_projects.total_revenue/currencies_beta.rate_one_eur as total_revenue_EUR ,
consultancy_projects.revenue/currencies_beta.rate_one_eur as sp_revenue_EUR , consultancy_projects.description as overview,
consultancy_projects.objective as objectives, consultancy_projects.activities
FROM consultancy_projects p, contacts c, currencies_beta k LIMIT 10) FROM `consultancy_projects`, `contacts`, `currencies_beta` UNION
(SELECT consultancy_projects.project_no, consultancy_projects.name, contacts.name as client,
consultancy_projects.total_revenue/currencies_beta.rate_one_eur as total_revenue_EUR ,
consultancy_projects.revenue/currencies_beta.rate_one_eur as sp_revenue_EUR , consultancy_projects.description as overview,
consultancy_projects.objective as objectives, consultancy_projects.activities
FROM consultancy_projects p, contacts c, currencies_beta k LIMIT 10)
</p>
<p>
    <strong>MySQL said: </strong><a href="./url.php?url=https%3A%2F%2Fdev.mysql.com%2Fdoc%2Frefman%2F5.5%2Fen%2Ferror-messages-server.html" target="mysql_doc"><img src="themes/dot.gif" title="Documentation" alt="Documentation" class="icon ic_b_help" /></a>
</p>
<code>#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'FROM `consultancy_projects`, `contacts`, `currencies_beta` UNION<br />
(SELECT consul' at line 5</code><br/></div>
```

**If I run any other query which does not contain union selects it works as expected!**

### Server configuration
**Operating system**: Debian 8.5
**Web server:** Apache 2.4.10
**Database:** MySQL 5.5.49
**PHP version:** 5.6.22
**phpMyAdmin version:** 4.6.4

**Comment by devenbansod** [comment]
I believe this is a duplicate of #12579, has been fixed and would be a part of next bug-fix release 4.6.5 .
You can try out the above query (and the export) with your data at https://demo.phpmyadmin.net/QA_4_6/?pma_username=root to confirm if the issue is indeed fixed or not.

**Comment by lukasg** [comment]
Thanks for the quick reply! I tested it on the demo system (db: lukasg_test) but I get the exact same result as on my 4.6.4 version.
